Instruments/config/cross_section.cfg

```diff
@@ -0,0 +1,59 @@
+DYJets: DYJetsToLL_M-50=6077.22 ref="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#DY_Z" unc="+-1.49 (integration) +- 14.78 (pdf) +- 2% (scale)"
+
+DYJets_M-10to50: DYJetsToLL_M-10to50_ext1=18610 ref="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#DY_Z" unc="unknown"
+
+EWK: EWKWMinus2Jets_WToLNu_M-50=20.25 EWKWPlus2Jets_WToLNu_M-50=25.62 EWKZ2Jets_ZToLL_M-50=3.987 ref="HTT AN-19-109" unc="unknown"
```

Comment:
- do not create multi-value entries
- keep property names simple e.g. "EWKWMinus2Jets_WToLNu_M-50: xs=20.25"